mono/metadata/class-init.c

```diff
@@ -1563,7 +1563,7 @@ count_virtual_methods (MonoClass *klass)
 		mcount = mono_class_get_method_count (klass);
 		for (i = 0; i < mcount; ++i) {
 			flags = klass->methods [i]->flags;
-			if (flags & METHOD_ATTRIBUTE_VIRTUAL)
+			if (!(flags & METHOD_ATTRIBUTE_STATIC))
```

Comment:
This doesn't seem right for normal classes.  I think this will count non-`static` non-`virtual` methods of classes.